src/python/WMCore/MicroService/Unified/MSTransferor.py

```diff
@@ -597,7 +597,7 @@ def makeTransferRequest(self, wflow):
                 # secondary already in place
                 continue
 
-            if wflow.getPURSElist():
+            if wflow.getPURSElist() and not wflow.isRelVal():
```

Comment:
@todor-ivanov thanks for your review. I was missing this case, where the relval has a pileup dataset and it was entering in this block, thus not bypassing the quotas. Nothing else changed with my commit (squashed) from 2min ago.